### ISSUE TITLE
Same terms are loaded via AJAX again and again

### ISSUE THREAD

**Issue by theo-gk** [superspeedyplugins/issue-tracker#125]
I have a lot of attribute terms, so I use the option "Items per page (Ajax)" with value 15, to load terms in batches. 
But when I scroll to the bottom, the same options are loaded with AJAX, and not the next batch as expected.
I noticed this bug on version 5.41. I don't know in which version it was introduced, but it was working fine in 5.29, before updating to 5.41.

**Steps to reproduce the behavior**
1. Add an attribute filter widget, setting the **Items per page (Ajax)** value to 15 -> [Settings screenshot](https://prnt.sc/u6urGWiSIW74)
The attribute selected for filtering should have more than 15 active terms for the current archive page.
3. Scroll down inside the widget container all the way to the bottom, to make the next terms appear
4. The terms which appear are wrong. The same terms are fetched again, instead of the next batch.

**Screenshots**
[Top of the list](https://prnt.sc/hSKBGJ31hUwe)
[Next terms loaded via AJAX](https://prnt.sc/5bUDfWfdigi_)

**Video**
[Short video with problem](https://www.loom.com/share/33320be2891a442f85b694b384b597a1)

**Devices (Problem seems unrelated to devices):**
 - OS: Windows, Android
 - Browser: All of them

**Additional context**
- I also have Scalability pro installed
- [SSF advanced settings](https://prnt.sc/WQfJOVFR6lWf)
- SSF constants overrides
  - define('SSF_MAKE_FILTERS_BE_FILTERED_BY_SEARCH', true);
  - define('SSF_REMOVE_SHOP_FROM_BREADCRUMBS', true);
  - define('SSF_SHOW_PARENTS_PER_TERM_IN_BREADCRUMBS', true);

I will appreciate if you could have a look into this soon, because my filters are basically not usable right now.
Thank you!

**Comment by dhilditch** [comment]
Thank you for the beautiful and concise bug description, the fix has been implemented in 5.43 which will now be available for you to update to.

**Comment by theo-gk** [comment]
Hello, I'm afraid I have to re-open this. 

I have updated to version 5.57 and when I scroll, I don't get the next page. 
Before your fix, I was always getting page 1 again and again. Now I don't get anything, nothing loads via AJAX, it's just page 1 visible. [Screenshot](https://prnt.sc/RKNIjHSE84H6)

I have not changed any settings, or modified anything else. I just updated to the newer version. 

Thank you
